diff --git a/discounts.py b/discounts.py
index 1a0a13a..1f45b47 100644
--- a/discounts.py
+++ b/discounts.py
@@ -21,7 +21,7 @@ app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 def hello():
     return Response({'Hello from Discounts!': 'world'}, mimetype='application/json')
 
-@app.route('/discount', methods=['GET', 'POST'])
+@app.route('/coupon', methods=['GET', 'POST'])
 def status():
     if flask_request.method == 'GET':
 
